### DISCUSSION TITLE
SomfyPlus `*** Warning.  Can't find SPI Device with Chip Select Pin=33`

### DISCUSSION THREAD

**Discussion by ViViDboarder** [HomeSpan/HomeSpan#681] [General]
Hello. I'm new to HomeSpan and Arduino, but not totally new to similar projects.

I already got the basic light bulb to connect and use the terminal. I've soldered up all the leads for SomfyPlus as per the diagram and I'm getting the following warning:

```
*** Warning.  Can't find SPI Device with Chip Select Pin=33
```

I've used my multimeter to see if there is a connection from the all ends of my connection from pin 33 all the way to the final CS lead on the Featherwing, and it seems to be connected. I've tried other pins on the ESP32 and got corresponding errors (same except the error mentioned the new pin). I'm not sure what else to try here or if I perhaps have a defective Featherwing.

**Comment by HomeSpan** [comment]
What chip are you using, and which featherwing (the exact same ones from Adafruit?)

**Reply by ViViDboarder** [reply]
Yea. I used the ones linked from the readme.

**Comment by HomeSpan** [comment]
Can you post pictures of the front and back of the your ESP32 and the RFM69 Featherwing?

**Reply by ViViDboarder** [reply]
Thanks for trying to help. Here's the pictures of the hardware.

![IMG_5305](https://github.com/HomeSpan/HomeSpan/assets/137025/8d25d22a-08cd-45c7-a95f-b3fa17326dfd)
![IMG_5303](https://github.com/HomeSpan/HomeSpan/assets/137025/67a21eb0-8401-43b3-98d2-8d6433fe0afc)
![IMG_5304](https://github.com/HomeSpan/HomeSpan/assets/137025/a15be463-c327-4f32-8384-da01a9a67a7c)
![IMG_5302](https://github.com/HomeSpan/HomeSpan/assets/137025/1a8225cd-2cea-4a98-8765-d11c03754eea)

I can probably re-upload and mark out where I tested with my DMM, if that would help.

**Comment by ViViDboarder** [comment]
So today, after removing and putting all the wires back in the headers, I'm still seeing the same error, but the TX LED is blinking. I don't remember seeing that last time. I'm not sure if that means that it must be communicating with the SPI Chip Select pin or not, but it's not pairing to blinds anyway.

**Comment by HomeSpan** [comment]
Unfortunately the label on the RFM that contains the frequency is blank - are there any markings that indicate the frequency anywhere?  The reason I ask is that there have been mix-ups between the two RFM modules Adafruit offers, and we want to make sure they shipped you the right one.

**Reply by ViViDboarder** [reply]
Huh. Didn't think of that. No, I don't see any markings anywhere.

**Comment by HomeSpan** [comment]
Sorry, I just realized in your photo that you did not solder headers to the RFM.  You should be plugging the RFM featherwing directly into the ESP32 featherwing with all the headers.

**Reply by ViViDboarder** [reply]
Yea, I used wire instead because I wanted to make it easier for me to double check things before using the headers. I can give that a go though.
